src/foam/core/AbstractFObjectPropertyInfo.java

```diff
@@ -9,7 +9,5 @@
 public abstract class AbstractFObjectPropertyInfo
   extends AbstractObjectPropertyInfo
 {
-  //  public int compareValues(FObject o1, FObject o2) {
-  //    return o1.compareTo(o2);
-  //  }
+  public abstract String of();
```

Comment:
DAO's use getOf(), so you could rename it to that for consistency, but I can live with either.